components/src/core/hero/hero.component.spec.ts

```diff
@@ -13,11 +13,11 @@ describe('HeroComponent', () => {
         }).compileComponents();
     }));
 
-    it(`Div element should have class as 'wrapper'`, () => {
+    it(`root element should have class as 'root'`, () => {
         const fixture = TestBed.createComponent(HeroComponent);
         const heroComponent = fixture.debugElement;
         const wrapperDiv: HTMLElement = heroComponent.query(By.css('div')).nativeElement;
-        expect(wrapperDiv.getAttribute('class')).toEqual('wrapper');
+        expect(wrapperDiv.getAttribute('class')).toEqual('root');
```

Comment:
Unless we are protecting our class names from changes at this stage, I would say they provide no value. 

I think it would be a good idea when we release R16 that we treat our class names the same way as we do our @Input names.  We should not be changing them very often.

Reply:
yes, if we are expecting users to do styles based on these classes, they should almost never be changed...any changes would be considered breaking

Reply:
are you also planning to prefix these classes (e.g., `pxblue-root`) like we discussed previously?

Reply:
I think it's a good idea, I'll prefix all all the class names.